internal/lsp/server.go

```diff
@@ -921,3 +921,10 @@ func getCompletionClientCapabilities(params *lsproto.InitializeParams) *lsproto.
 	}
 	return params.Capabilities.TextDocument.Completion
 }
+
+func getDefinitionClientCapabilities(params *lsproto.InitializeParams) *lsproto.DefinitionClientCapabilities {
```

Comment:
At some point I want to try and consolidate this with helpers on `ctx`, since everything seems to use a different mechanism to control what they send (no action needed now)